region/norfair/west/Hi Jump Energy Tank Room.json

```diff
@@ -240,10 +338,72 @@
         {"or": [
           "canWalljump",
           "h_canFly",
-          "canUseFrozenEnemies"
+          "canUseFrozenEnemies",
+          {"and": [
+            "HiJump",
+            "canSpringBallJumpMidAir"
+          ]}
         ]}
       ]
     },
+    {
+      "link": [2, 1],
+      "name": "G-Mode through Bomb Blocks",
+      "entranceCondition": {
+        "comeInWithGMode": {
+          "mode": "any",
+          "morphed": true
+        }
+      },
+      "requires": [
+        "h_canArtificialMorphMovement",
+        "canOffScreenMovement"
+      ],
+      "note": "Overload PLMs using scroll blocks in the morph tunnel. The camera will not scroll to the left side of the room."
+    },
+    {
+      "link": [2, 1],
+      "name": "G-Mode Overload PLMs with the Left Item",
+      "entranceCondition": {
+        "comeInWithGMode": {
+          "mode": "direct",
+          "morphed": true
+        }
+      },
+      "requires": [
+        {"itemNotCollectedAtNode": 4},
+        "canRiskPermanentLossOfAccess",
+        "h_canArtificialMorphPowerBomb",
+        "canOffScreenMovement"
+      ],
+      "note": [
+        "Placing a Power Bomb a 0.5 to 1 tile away from the item plinth to bounce Samus onto it (where the right item usually is) while also overloading PLMs as the explosion hits the left item.",
+        "Wait for the explosion to completely finish before touching the crumble blocks or they will remain solid.",
+        "The camera will not scroll to the left side of the room."
+      ],
+      "devNote": "FIXME: This item will be remote acquired, but it is ignored here since the strat is canRiskPermanentLossOfAccess."
```

Comment:
The right item would be remote collected, but isn't a requirement for the strat.  But its still an unrepresented something.

Reply:
I could just fix it and add an obstacle if you think its important.

We would really need a way to distinguish when an item is optional to collect or forced to be collected if we were to use this type of strat in the logic.

Reply:
I don't think its important right now.